src/components/travel-search-filter/index.tsx

```diff
@@ -0,0 +1,255 @@
+import { getTextForLanguage } from '@atb/translations/utils';
+import { useTranslation } from '@atb/translations';
+import { ChangeEvent, useState } from 'react';
+import style from './travel-search-filter.module.css';
+import { getTransportModeIconPath } from '@atb/components/travel-search-filter/utils';
+import { MonoIcon } from '@atb/components/icon';
+import { TransportModeFilterOptionWithSelectionType } from '@atb/components/travel-search-filter/types';
+import { Typo } from '@atb/components/typography';
+
+const defaultTravelSearchFilter: TransportModeFilterOptionWithSelectionType[] =
```

Comment:
How should we handle this initial state? I assume it needs to be passed as prop, but should we have it hard coded or set up firestore for this and other stuff?

Reply:
I'm not sure if we should use Firestore or not, but I just want to comment that there is some infra work needed if we want to use Firestore. Maybe we could hard code it for now and potentially connect to Firestore in another PR?

Reply:
The Firestore addition would happen in a later PR regardless. More so if the placement of the state should be temporary or not.